x/ecocredit/module/module.go

```diff
@@ -48,18 +44,24 @@ func (a AppModule) ExportGenesis(sdk.Context, codec.JSONMarshaler) json.RawMessa
 }
 
 func (a AppModule) GetTxCmd() *cobra.Command {
-	return nil
+	return client.TxCmd()
 }
 
-func (a AppModule) GetQueryCmd() *cobra.Command { return client.QueryCmd() }
+func (a AppModule) GetQueryCmd() *cobra.Command {
+	return client.QueryCmd()
+}
 
 func (a AppModule) RegisterGRPCGatewayRoutes(sdkclient.Context, *runtime.ServeMux) {}
 
 func (a AppModule) RegisterInvariants(sdk.InvariantRegistry) {}
 
 func (a AppModule) RegisterServices(cfg module.Configurator) {
-	ecocredit.RegisterMsgServer(cfg.MsgServer(), a.Srv)
-	ecocredit.RegisterQueryServer(cfg.QueryServer(), a.Srv)
+	key := sdk.NewKVStoreKey(ecocredit.ModuleName)
```

Comment:
The key cannot be created in `RegisterServices`. Then it won't get mounted with the multistore.